cfpb/cfgov-refresh
# Mega menu: Cache Wagtail version of menu

Adds caching for Wagtail version of mega menu so its page queries don't impact initial page load times.

## Additions

- New template fragment cache
- Caching for Wagtail version of menu

## Testing

1. Check out branch
2. Create cache table: `python cfgov/manage.py createcachetable default_fragment_cache`
3. Run the server with Django Debug Toolbar & caching enabled: `ENABLE_DEBUG_TOOLBAR=1 ENABLE_DEFAULT_FRAGMENT_CACHE=1 ./runserver.sh`
4. Turn on `WAGTAIL_MENU` flag
5. Load [http://localhost:8000/](http://localhost:8000/) and note the large number of queries in the debug toolbar
6. Reload [http://localhost:8000/](http://localhost:8000/)  and note the number of queries has decreased as the menu is loaded from cache
7. Save a change to a Menu Item.  Reload [http://localhost:8000/](http://localhost:8000/) and verify that the cache has been cleared and the updated menu item is displayed

## Checklist
